stdpopsim/catalog/homo_sapiens.py

```diff
@@ -426,7 +426,7 @@ class _BrowningAmerica(HomoSapiensModel):
         stdpopsim.Population(name="ASIA", description="Contemporary Asian population"),
         stdpopsim.Population(
             name="ADMIX", description="Ancient admixed population",
-            allow_samples=False),
+            sampling_time=-1),
```

Comment:
I think it'd be better to use ``None`` as the exceptional value here. Seems easier to read, somehow with ``sampling_time=None``. It's a bit more explicit, and bit less likely to happen by accident.

Reply:
Done

Reply:
done